exe/fil-tools/cmd/deals_store.go

```diff
@@ -54,8 +54,8 @@ var storeCmd = &cobra.Command{
 		}
 
 		file, err := os.Open(path)
-		defer file.Close()
 		checkErr(err)
+		defer file.Close()
```

Comment:
Bugfix in the wild.

Reply:
thx

Reply:
Although, I guess my thinking was that the file should be closed even if there is an error. In the case of an error, this change would cause the the program to exit before closing the file (not sure this is even a problem).

Reply:
Actually, it wouldn't because the file can't be considered open since the operation failed.

Making calls on return values from failed operations most probably will result in panics or other errors. When an operation fails, we can't consider return values (other than err) to be valid.